### PR TITLE
bonnie-prior 1302 nokogiri/loofah/rails-html-sanitizer fixes

### PR DESCRIPTION
**Pull request by c-monkey**
This PR fixes vulnerabilities found running `bundle exec bundle-audit check --update`.

----

:exclamation: **Must be updated before merge to point  `HDS` to mongoid5 branch and `h1mf2js`, `patientapi`, and `simplexml_parser` to bonnie-prior branch. These lines are in the Gemfile, commented-out.**
- [x] [HDS](https://github.com/projectcypress/health-data-standards/pull/566)
- [x] [hqmf2js](https://github.com/projecttacoma/hqmf2js/pull/54)
- [x] [patientapi](https://github.com/projecttacoma/patientapi/pull/20)
- [x] [simplexml_parser](https://github.com/projecttacoma/simplexml_parser/pull/43)

----

Pull requests into Bonnie Bundler require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [X] This pull request describes why these changes were made.
- [X] This PR is into the correct branch.
- [X] Internal ticket for this PR: https://jira.mitre.org/browse/BONNIE-1302
- [x] JIRA ticket links to this PR
- [X] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [X] Tests are included and test edge cases
- [X] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [X] Code coverage has not gone down and all code touched or added is covered.
    * In rare situations, this may not be possible or applicable to a PR. In those situations:
        1. Note why this could not be done or is not applicable here:
        2. Add TODOs in the code noting that it requires a test
        3. Add a JIRA task to add the test and link it here:
- [X] Automated regression test(s) pass

If JIRA tests were used to supplement or replace automated tests:
- [X] JIRA test links: N/A
- [X] Justification for using JIRA tests: N/A
- [X] JIRA tests have been added to sprint N/A

**Reviewer 1:**

Name: @hossenlopp
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [X] JIRA tests have been run and pass N/A
- [X] You agree with the justification for use of JIRA tests or have provided input on why you disagree N/A

**Reviewer 2:**

Name: @losborne
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [X] JIRA tests have been run and pass N/A
- [X] You agree with the justification for use of JIRA tests or have provided input on why you disagree N/A

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -120,13 +124,14 @@ GEM
     execjs (2.1.0)
     globalid (0.4.0)
       activesupport (>= 4.2.0)
-    highline (1.7.8)
+    highline (1.7.10)
     http-cookie (1.0.3)
       domain_name (~> 0.5)
     i18n (0.8.4)
     libv8 (3.16.14.11)
```
Comment (lukoktonos) [inline Gemfile.lock]: As with https://github.com/projecttacoma/patientapi/pull/20, this should be updated to `libv8 (3.16.14.19)`

**Review by lukoktonos — CHANGES_REQUESTED** [review]
Bundle audit and tests pass (skipped tests pass locally despite the CI breaking).

Only change requested is libv8 version, similarly to the patientapi repo.

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 language: ruby
 rvm:
-  - "2.3.4"
+  - "2.3.5"
```
Comment (hossenlopp) [inline .travis.yml:3]: bonnie-prior is still at 2.3.4.

**Review by lukoktonos — APPROVED** [review]

**Comment by c-monkey** [comment]
The skips are from errors that arose in this PR:
https://github.com/projecttacoma/bonnie_bundler/pull/112

**Review by hossenlopp — APPROVED** [review]
